### DISCUSSION TITLE
如果有来自个人或者企业对开源面对面的赞助的话，我们该如何分配

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#238] [Ideas]
在节目的[两周年](https://github.com/opensource-f2f/episode/issues/234)特别节目的录制过程中，我们聊到了一个话题——如果有来自个人或者企业对开源面对面的赞助的话，我们该如何分配。以下是我们交流通过的一些记录，也欢迎各位关注开源面对面的朋友们一起讨论。

@MikeyWei 建议把部分资金分配给做后期制作（剪辑等）的 contributor，核心考虑是这些后期制作的付出很难被明显、直接地看到
@wey-gu 建议把部分资金分配给所有参与节目制作的人员
@majinghe 建议找商业的设计把节目的 Logo 进行优化
@LinuxSuRen 没有想到具体的分配方案，但希望能实现节目集市化的发展，如果有可能的话，花到相关环节

另外，大家也一致认为会有一些公共的支出，例如：网站、域名、商标注册、周边制作等等。

考虑到很多企业会在年底制定次年的预算，我们会在年底时试着寻找一些潜在的赞助企业。

但是，我们认为的赞助不会是接商业单子的形式，不会在节目中插播广告之类的。对赞助者的感谢方式，大家可以讨论，并且最终会形成正式文字。

> 之前，在爱发电平台上，已经收到过[三笔来自不同朋友们的捐赠](https://afdian.net/a/opensource-f2f)，之后会统一管理。

**Comment by MikeyWei** [comment]
> 在节目的[两周年](https://github.com/opensource-f2f/episode/issues/234)特别节目的录制过程中，我们聊到了一个话题——如果有来自个人或者企业对开源面对面的赞助的话，我们该如何分配。以下是我们交流通过的一些记录，也欢迎各位关注开源面对面的朋友们一起讨论。
> 
> @MikeyWei 建议把部分资金分配给做后期制作（剪辑等）的 contributor，核心考虑是这些后期制作的付出很难被明显、直接地看到 @wey-gu 建议把部分资金分配给所有参与节目制作的人员 @majinghe 建议找商业的设计把节目的 Logo 进行优化 @LinuxSuRen 没有想到具体的分配方案，但希望能实现节目集市化的发展，如果有可能的话，花到相关环节
> 
> 另外，大家也一致认为会有一些公共的支出，例如：网站、域名、周边制作等等。
> 
> 考虑到很多企业会在年底制定次年的预算，我们会在年底时试着寻找一些潜在的赞助企业。
> 
> 但是，我们认为的赞助不会是接商业单子的形式，不会在节目中插播广告之类的。对赞助者的感谢方式，大家可以讨论，并且最终会形成正式文字。
> 
> > 之前，在爱发电平台上，已经收到过[三笔来自不同朋友们的捐赠](https://afdian.net/a/opensource-f2f)，之后会统一管理。

在 logo 的基础上，建议加上注册商标相关事宜，在当前的大环境下，做一些预防性的知识产权保护是由必要的。

**Reply by genedna** [reply]
公共支出是最好的，引起爭議的可能性最小

**Comment by shinyzhu** [comment]
赞成统一管理。分配在一些需要支出的公共费用上。

**Comment by wey-gu** [comment]
> @wey-gu 建议把部分资金分配给所有参与节目制作的人员

我的希望是如果 sponsorship 到一定程度，可以像基金会雇佣全职员工那样，以某种程度，“雇佣”一些人，让他们（i.e. Rick）可以更体面、从容地把更多时间放在节目的制作上。

虽然这是一个 hard way。

**Reply by LinuxSuRen** [reply]
这也是个 Long way。我了解到的一些基金会，只有非常少量的全职员工。如果真能沉淀出足够有价值的内容来，也非常期待有那一天。

**Comment by d4x1** [comment]
具体怎么花钱跟组织的目标以及有多少钱（收到的赞助规模）有关系。
不过钱多少对优先级的影响不是很大，我认为合理的优先级应该是：让组织长久的存在下去 > 把组织做大 > 其他（体面、全职员工），按照这个来花钱就行了。
持有不同目标的人对优先级的理解可能会不一样。

**Reply by LinuxSuRen** [reply]
广义上来讲，通常会希望把一件事做久、做大。不过，换个视角来看，做某事背后的动机、能解决的问题也很重要，如果确实我们做的事情是有意义的、能帮助到一部分群体的，那么是值得持久地做下去的。而是否要做大，可能会是一个结果而非目标，我比较倾向于把事情做好、做的更好，或者还是以传递价值、正能量为目标，做大还是保持现状可能是做好以后的结果、现象。

以上是个人浅见，一个健康、良好的组织内也应该有不同的声音、方向，只要大的方向是没问题就是好的。期待大家多多发表观点，讨论起来、行动起来。

**Reply by d4x1** [reply]
那还是需要把钱花在能帮助更多人上面：产出质量更高的内容（邀请更能带来正向帮助的人、节目质量更高等），扩大影响范围，触达更多的人群。

**Reply by LinuxSuRen** [reply]
是的，非常同意。
